main.py

```diff
@@ -122,12 +216,14 @@ def _is_video_tweet(embed: discord.Embed) -> bool:
     if not is_twitter_embed:
         return False
     if embed.image.url is None:
-        return True
+        return False
 
     # native video embed now uses a thumbnail instead of broken video
     # URL begins with https://pbs.twimg.com/ext_tw_video_thumb/
+    # or https://pbs.twimg.com/tweet_video_thumb/
     contains_video_thumbnail = "ext_tw_video_thumb" in embed.image.url
-    return contains_video_thumbnail
+    contains_gif_thumbnail = "tweet_video_thumb" in embed.image.url
+    return contains_video_thumbnail or contains_gif_thumbnail
 
 
 def _is_sensitive_tweet_embed(embed: discord.Embed) -> bool:
```

Comment:
This should possibly also move to using fxtwitter API, but as per Discord conversation, it makes sense ("more likely to be reflective of [discord embed behavior]")